check_urls.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+from concurrent import futures
+import multiprocessing as mp
+import os
+
+import requests
+import urllib3
+
+
+# NOTE: Ignore security hazard since certs SHOULD be trusted
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Configurable variables
+URL_MATCH = 'https?:\/\/[a-zA-Z0-9\.\-]+(html|\/)[=a-zA-Z0-9\_\/\?\&\-]+'
```

Comment:
The last portion of the regex `[=a-zA-Z0-9\_\/\?\&\-]+` should be `[=a-zA-Z0-9\_\/\?\&\.\-]+` since it missed the `.` thereby ignoring websites that ended with `.html`.